web-console/src/lib/components/streaming/inspection/InspectionTable.tsx

```diff
@@ -99,16 +99,22 @@ const useInspectionTable = ({ pipeline, name }: InspectionTableProps) => {
       return
     }
     const controller = new AbortController()
-    const url = new URL(
-      OpenAPI.BASE +
-        '/v0' +
-        '/pipelines/' +
-        pipeline.descriptor.pipeline_id +
-        '/egress/' +
-        name +
-        '?format=csv&query=neighborhood&mode=watch'
-    )
-    updateTable(url, neighborhood, setRows, setLoading, relation, controller).then(
+    updateTable(
+      [
```

Comment:
I am a little puzzled by this signature.
The first argument is an array of ANY?
Can this be stronger-typed?

Reply:
My type-checker works as expected:
![image](https://github.com/feldera/feldera/assets/8537259/361945bc-4524-482b-8014-bf8666b29cda)
enabled by 
![image](https://github.com/feldera/feldera/assets/8537259/92d6f186-ec3d-45f5-b53f-4af9c61477e6)

This change is final, it is designed to follow dependency injection principle (I think it's a good fit here)
So the only thing to be updated later is, in this case, implementation of `updateTable` function

Reply:
![image](https://github.com/feldera/feldera/assets/8537259/859a0754-91fd-4754-becd-0809f7a7fce8)

Reply:
ah, it's a tuple, not an array